README.md

```diff
@@ -20,8 +20,8 @@
 | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
 | Travis CI  | [.travis.yml](https://github.com/mmzakhar/oneapi-ci/blob/master/.travis.yml) | [![Alt text](https://travis-ci.com/mmzakhar/oneapi-ci.svg?branch=master)](https://travis-ci.com/mmzakhar/oneapi-ci)|+|+|+|+|+|+|+|+|+|+|+|
-| Circle CI  | [.circleci/config.yml](https://github.com/mmzakhar/oneapi-ci/blob/master/.circleci/config.yml) | [![Alt text](https://circleci.com/gh/mmzakhar/oneapi-ci.svg?style=svg)](https://circleci.com/gh/mmzakhar/oneapi-ci)|-|-|-|+|+|+|+|+|+|-|-|
-| AppVeyor  | [.appveyor.yml](https://github.com/mmzakhar/oneapi-ci/blob/master/.appveyor.yml) | [![Alt text](https://ci.appveyor.com/api/projects/status/y06fiwtls22x7475?svg=true)](https://ci.appveyor.com/project/mmzakhar/oneapi-ci)|+|+|+|+|+|+|-|-|-|+|+|
+| Circle CI  | [.circleci/config.yml](https://github.com/oneapi-src/oneapi-ci/blob/master/.circleci/config.yml) | [![Build status](https://circleci.com/gh/mmzakhar/oneapi-ci.svg?style=svg)](https://circleci.com/gh/mmzakhar/oneapi-ci)|-|-|-|+|+|+|+|+|+|-|-|
+| AppVeyor  | [.appveyor.yml](https://github.com/oneapi-src/oneapi-ci/blob/master/.appveyor.yml) | [![Alt text](https://ci.appveyor.com/api/projects/status/y06fiwtls22x7475?svg=true)](https://ci.appveyor.com/project/mmzakhar/oneapi-ci)|+|+|+|+|+|+|-|-|-|+|+|
 | GitLab CI  | [.gitlab-ci.yml](https://github.com/mmzakhar/oneapi-ci/blob/master/.gitlab-ci.yml) | [![Alt text](https://gitlab.com/mmzakhar/oneapi-ci/badges/master/pipeline.svg)](https://gitlab.com/mmzakhar/oneapi-ci)|+|+|+|+|+|+|-|-|-|-|-|
-| GitHub Actions | [.github/workflows/build_all.yml](https://github.com/mmzakhar/oneapi-ci/blob/master/.github/workflows/build_all.yml) | [![Alt text](https://github.com/mmzakhar/oneapi-ci/workflows/build_all/badge.svg)](https://github.com/mmzakhar/oneapi-ci)|+|+|+|+|+|+|+|+|+|+|+|
-| Azure Pipelines| [.azure-pipelines.yml](https://github.com/mmzakhar/oneapi-ci/blob/master/.azure-pipelines.yml) | [![Build Status](https://dev.azure.com/maximmzakharov/oneapi-ci/_apis/build/status/mmzakhar.oneapi-ci?branchName=master)](https://dev.azure.com/maximmzakharov/oneapi-ci/_build?definitionId=2)|+|+|+|+|+|+|+|+|+|+|+|
+| GitHub Actions | [.github/workflows/build_all.yml](https://github.com/mmzakhar/oneapi-ci/blob/master/.github/workflows/build_all.yml) | [![Build status](https://github.com/mmzakhar/oneapi-ci/workflows/build_all/badge.svg)](https://github.com/mmzakhar/oneapi-ci)|+|+|+|+|+|+|+|+|+|+|+|
+| Azure Pipelines| [.azure-pipelines.yml](https://github.com/oneapi-src/oneapi-ci/blob/master/.azure-pipelines.yml) | [![Build Status](https://dev.azure.com/robertscohn/oneapi-ci-mirror/_apis/build/status/robertscohn.oneapi-ci-mirror?branchName=master)](https://dev.azure.com/robertscohn/oneapi-ci-mirror/_build)|+|+|+|+|+|+|+|+|+|+|+|
```

Comment:
For some reason the Badge URL for Azure Pipelines (https://dev.azure.com/robertscohn/oneapi-ci-mirror/_apis/build/status/robertscohn.oneapi-ci-mirror?branchName=master) leads to a "set up now" image rather than build status image.